grafana/grafyaml/dashboard-nodepool.yaml

```diff
@@ -262,9 +270,26 @@ dashboard:
           seriesOverrides:
             - alias: Max
               stack: False
+        - title: Test Nodes (LON)
+          type: graph
+          span: 3
+          stack: true
+          tooltip:
+            value_type: individual
+          leftYAxisLabel: "nodes"
+          targets:
+            - target: alias(stats.gauges.nodepool.provider.pubcloud-lon.nodes.building, 'Building')
+            - target: alias(stats.gauges.nodepool.provider.pubcloud-lon.nodes.ready, 'Available')
+            - target: alias(stats.gauges.nodepool.provider.pubcloud-lon.nodes.in-use, 'In Use')
+            - target: alias(stats.gauges.nodepool.provider.pubcloud-lon.nodes.used, 'Used')
+            - target: alias(stats.gauges.nodepool.provider.pubcloud-lon.nodes.deleting, 'Deleting')
+            - target: alias(stats.gauges.nodepool.provider.pubcloud-lon.max_servers, 'Max')
+          seriesOverrides:
+            - alias: Max
+              stack: False
         - title: Test Nodes (ORD)
           type: graph
-          span: 4
+          span: 3
```

Comment:
Whats `span` here?

Reply:
It's the width of the graph, spanning x columns. Given that we're adding a new one they need to be reduced to fit into the same space.

Reply:
Makes sense, thanks 👍